src/modules/logger/logged_topics.cpp

```diff
@@ -70,7 +70,7 @@ void LoggedTopics::add_default_topics()
 	add_optional_topic("flaps_setpoint", 1000);
 	add_optional_topic("flight_phase_estimation", 1000);
 	add_optional_topic("fuel_tank_status", 10);
-	add_topic("gimbal_manager_set_attitude", 500);
+	add_topic("gimbal_manager_set_attitude");
```

Comment:
Is this change just for testing or a permanent change? Then please explain motivation in the PR description, and/or make separate commit.